test/scenarios.py

```diff
@@ -8353,4 +8353,73 @@
             ),
         },
     ),
+    Scenario(
+        "scenario43",
+        "Sucesfully transfer the file. Then restart the receiver. Expect no events on receiver reconnection",
+        {
+            "DROP_PEER_REN": ActionList(
+                [
+                    action.Start("DROP_PEER_REN"),
+                    action.NewTransfer("DROP_PEER_STIMPY", ["/tmp/testfile-small"]),
+                    action.Wait(
+                        event.Queued(
+                            0,
+                            {
+                                event.File(
+                                    FILES["testfile-small"].id,
+                                    "testfile-small",
+                                    1048576,
+                                ),
+                            },
+                        )
+                    ),
+                    action.Wait(event.Start(0, FILES["testfile-small"].id)),
+                    action.Wait(
+                        event.FinishFileUploaded(
+                            0,
+                            FILES["testfile-small"].id,
+                        )
+                    ),
+                    action.NoEvent(2),
```

Comment:
nitpick: this test might use a bit bigger NoEvent argument, especially when parallel executing it might be that receiver will do it's action slower and the sender will already succeed at waiting 2 seconds with no events

Reply:
done